ci-operator/jobs/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main-periodics.yaml

```diff
@@ -91,6 +91,87 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 25 10 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: jboss-fuse
+    repo: camel-k-test-container
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: camel-k-ocp4.14-lp-rosa
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-rosa-camel-k-interop-aws-rosa-sts
```

Comment:
We need to include the slack notification channel here if there is one for this product team.